app/src/main/java/org/simple/clinic/recentpatientsview/RecentPatientsViewController.kt

```diff
@@ -60,7 +61,10 @@ class RecentPatientsViewController @Inject constructor(
 
     val updateRecentPatients = recentPatientsStream
         .map { it.map(::recentPatientItem) }
-        .map { it + SeeAllItem }
+        .withLatestFrom(patientConfig)
+        .map { (recentPatients, config) ->
+          maybeAddSeeAllButton(recentPatients, config)
```

Comment:
We can actually inline this code since this method is not being reused.
Also the scope of `maybeAddSeeAllButton` method is limited to `showRecentPatients()`.

Reply:
Inlined